db/schemas/schemas.py

```diff
@@ -11,7 +12,47 @@ class AssessmentRecordMetadata(SQLAlchemyAutoSchema):
 
     class Meta:
         model = AssessmentRecord
-        exclude = ["jsonb_blob", "application_json_md5"]
 
     workflow_status = Enum(Status)
     language = Enum(Language)
+
+
+ASSESSOR_TASK_LIST_METADATA_FIELDS = (
+    "short_id",
+    "project_name",
+    "workflow_status",
+    "jsonb_blob",
+    "fund_id",
+    "round_id",
+)
+
+
+class AssessorTaskListMetadata(AssessmentRecordMetadata):
+    """AssessorTaskListMetadata The marshmallow class used to turn SQLAlchemy
+    rows into json for return in http responses.
+
+    Fields are:
+    - project_name
+    - project_reference (short_id)
+    - workflow_status
+    - date_submitted
+    - fund_id
+    - round_id
+    """
+
+    class Meta:
+        model = AssessmentRecord
+        exclude = tuple(
+            field
+            for field in AssessmentRecord.__table__.columns.keys()
+            if field not in ASSESSOR_TASK_LIST_METADATA_FIELDS
+        )
+
+    @post_dump
+    def discard_irrelevant_fields(self, item, many, **kwargs):
+        """Discard irrelevant fields from the jsonb_blob."""
+        item["date_submitted"] = item["jsonb_blob"]["date_submitted"]
+        item["project_reference"] = item["short_id"]
+        del item["short_id"]
+        del item["jsonb_blob"]
+        return item
```

Comment:
Why are we doing this?

Reply:
NB: see key and field here https://marshmallow.readthedocs.io/en/stable/marshmallow.fields.html#marshmallow.fields.Field . This might be a cleaner way of doing this. Also, we extract information from the json blob on insert so we don't have to do it here

Reply:
I'll take a deeper look at this tomorrow

Reply:
Have tried to utilize fields now and removed the ugly `post_dump` stuff.  If you know a better way to do this please let me know, admittedly not too familiar with this library.